data/nlu/nlu.md

```diff
@@ -529,6 +530,8 @@
 - what's the difference between [rasa](product) and [rasa x](product)
 - whats the diff between [rasa](product) and [rasa x](product)
 - when should i use [rasa](product) and when should i use [rasa x](product)
+- what is the difference between[ rasa open source](company)[rasa open source](product:rasa) and [rasa x](company)[rasa x](product:x)
```

Comment:
what is happening here?

Reply:
Oi boi. I referenced this as a challenge when reviewing convos - if you don't explicitly remove incorrect entities you end up in this situation (after adding correct ones). This PR is actually not relevant anymore because I merged the changes into the `use_response_selector` branch

Reply:
Sorry, I didn't request your review - just wanted to see what the PR would look like. not sure how that happened